src/schema/users.ts

```diff
@@ -1176,6 +1192,50 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
 
       return streak;
     },
+    streakRecover: async (
+      _,
+      __,
+      ctx: AuthContext,
+    ): Promise<StreakRecoverQueryResult> => {
+      const { userId } = ctx;
+
+      const streak = await ctx.con.getRepository(UserStreak).findOneBy({
+        userId,
+      });
+      const { currentStreak } = streak;
+      const currentStreakIsTooLongToRecover = currentStreak > 1;
```

Comment:
currentStreakIsTooLongToRecover - what do we mean by it?

Reply:
If user have a current streak of 2 is no possible to recover the old one because at least 2 days have passed since the loss. Maybe I can rename with `currentStreakIsTooLongForRecover` just because the check is "is the current streak too long for trying to recover the old one?". 

But thinking about this I think that if the currentStreak is > 1 we should not have the redis key as mentioned [here](https://github.com/dailydotdev/daily-api/pull/2156#discussion_r1726622461), so we can even remove the check. WDYT?

Reply:
> But thinking about this I think that if the currentStreak is > 1 we should not have the redis key as mentioned https://github.com/dailydotdev/daily-api/pull/2156#discussion_r1726622461, so we can even remove the check. WDYT?

Yeah with that I think we should be safe, though just since it is here I don't mind keeping it anyways - maybe we can say `timeForRecoveryPassed` to explain the purpose of the check.